src/main/java/org/broadinstitute/hellbender/tools/spark/sv/LocalAssemblyContig.java

```diff
@@ -0,0 +1,95 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import java.io.Serializable;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Represents a collection of relevant information on one locally assembled contig from short reads.
+ * TODO: should we move AlignmentRegion class here or keep it as top level class?
+ */
+class LocalAssemblyContig implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    /**
+     * This contig was assembled based on FASTQ records collected by {@link FindBadGenomicKmersSpark} and numbered by it.
+     */
+    final long assemblyID;
+
+    /**
+     * This together with {@link #assemblyID} completely determines which contig we are talking about.
+     */
+    final String contigID;
+
+    final String seq;
+
+    /**
+     * Semi-qual score of bases that's available from fermi-lite but not from SGA.
+     * May be {@code null} if assembler doesn't provide this feature
+     */
+    final byte[] quals;
+
+    /**
+     * Read count for the contigs.
+     * TODO: fermi-lite provides support for this, not SGA
+     */
+    final int readCountSupport;
+    static final int READ_COUNT_SUPPORT_DEFAULT = -1;
```

Comment:
Yes, was simply falling in the trap of primitive-bias and thought about this null-equivalent for it.